resources/translations/de_DE/infill/infill_wipe_dist.md

```diff
@@ -0,0 +1,11 @@
+Wipe-Abstand der Füllung
+====
+Bei dieser Einstellung bewegt sich die Düse ein wenig weiter als das Ende der Fülllinie. Ziel dieses kleinen Fahrwegs ist es, das Material auf der daneben liegenden Wand abzuwischen. Dadurch verbindet sich die Fülllinie besser mit der Wand.
+
+![Eine Visualisierung der Füllungsüberlappung und des Wischabstands](../images/infill_overlap.svg)
+
+Dieser Fahrweg macht das Objekt zwar stärker, hat aber den großen Nachteil, dass er durch die Wand geführt wird, was eine sichtbare Spur auf der Außenseite des Drucks hinterlässt. Dadurch scheint das Füllmuster stärker durch die Außenhaut hindurch.
+
+Dies gilt nur für die Enden der Fülllinien. Wenn Sie zusammenhängende Fülllinien verwenden, gibt es viel weniger Fülllinienenden.
+
+Übersetzt mit www.DeepL.com/Translator (kostenlose Version)
```

Comment:
Is it? Please do go through the translations then. Although I think you did make corrections, at least in this article.